src/platforms/generic_unix/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required (VERSION 2.6)
+project (libAtomVMPlatformGenericUnix)
+
+include_directories(${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_SOURCE_DIR}/src/libAtomVM)
+
+set(HEADER_FILES
+    mapped_file.h
+    udpdriver.h
+)
+set(SOURCE_FILES
+    sys.c 
+    mapped_file.c 
+    udpdriver.c
+)
+
+set(
+    PLATFORM_LIB_SUFFIX
+    ${CMAKE_SYSTEM_NAME}-${CMAKE_SYSTEM_PROCESSOR}
+)
+
+if(CMAKE_COMPILER_IS_GNUCC)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -pedantic -Wextra -ggdb")
+endif()
+
+add_library(libAtomVM${PLATFORM_LIB_SUFFIX} ${SOURCE_FILES} ${HEADER_FILES})
```

Comment:
I cannot understand why you need to add headers here, on linux it builds anyway without it, is it required on any other platform?

Reply:
oh, good point, thank you for pointing this out, I didn't realize this, I'll check it too :)